.gitignore

```diff
@@ -142,4 +142,9 @@ build.out
 # Azure Project
 csx/
 *.GhostDoc.xml
-pingme.txt
\ No newline at end of file
+pingme.txt
```

Comment:
Why was this added

Reply:
Just miss it in VS ..
BTW this SDK will be used in the next Powershell release? Any idea when this should be?